fah_xchem/analysis/diffnet.py

```diff
@@ -243,13 +267,17 @@ def combine_free_energies(
     #                   )
     #
     for compound in compounds:
-        pIC50 = compound.metadata.experimental_data.get("pIC50")
-
         # Skip compounds with no experimental data
-        if pIC50 is None:
+        if "g_exp" not in compound.metadata.experimental_data:
+            continue
+
+        # DEBUG: Use only one reference compound
+        if compound.metadata.compound_id != "VLA-UCB-50c39ae8-2":
```

Comment:
Whoops---leaving this in was a mistake! This was critical when I was debugging the issues with `arsenic` giving incorrect results. We should remove this, but this will also make it more difficult to compare the errors in absolute free energies, since those will include multiple reference compounds once this is removed. However, my focus will now be on microstate transformation errors since we now have correctly annotated microstates.

Reply:
@jchodera this one in particular had me concerned; can you elaborate on the intention here?

Reply:
Ah cool, thanks! Sorry, needed to refresh my page here.